apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts

```diff
@@ -28,6 +28,75 @@ export const AssessmentRowSchema = AssessmentSchema.merge(
 });
 type AssessmentRow = z.infer<typeof AssessmentRowSchema>;
 
+export function InstructorInstanceAdminLti13NoInstances({
+  resLocals,
+  ltiInstances,
+  institutionName,
+}: {
+  resLocals: Record<string, any>;
+  ltiInstances: Lti13Instance[];
+  institutionName: string;
+}): string {
+  const content = html`
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white d-flex align-items-center">
+        <h1>Integrations with other learning systems</h1>
+      </div>
+      <div class="card-body">
+        <p>
+          PrairieLearn supports integrations with learning systems to pass status and grades between
+          them. No integrations have been configured yet for this course instance, but when that
+          happens you can configure them there.
+        </p>
+
+        ${ltiInstances.length === 0
+          ? html`
+              <p>
```

Comment:
```suggestion
              <p class="mb-0">
```

Reply:
I don't mind the extra space around the <p> and <ul> tags. Is there a compelling reason to not have a bottom margin on these?

Reply:
No reason other than it looks much nicer to have even spacing on all sides.